client/plots/barchart.js

```diff
@@ -106,7 +106,11 @@ export class Barchart {
 					vocabApi: this.app.vocabApi,
 					numericEditMenuVersion: this.opts.numericEditMenuVersion,
 					defaultQ4fillTW: term0_term2_defaultQ,
-					getDisplayStyle: () => (this.settings.unit == 'log' ? 'none' : '')
+					getDisplayStyle: () => (this.settings.unit == 'log' ? 'none' : ''),
+					processConfig: () => {
+						//Prevent log scale shown in overlay
+						if (this.settings.unit == 'log') this.settings.unit == 'abs'
+					}
```

Comment:
This is incorrect. The option should be provided with the input opts in the screenshot below. You can see that `opts.processConfig()` is used in `setTermsInput()` callback line 718 in `controls.config.js`. 

![Screenshot 2024-11-05 at 10 01 35 AM](https://github.com/user-attachments/assets/c898ec60-84c0-4bc1-bb81-a85ca6d3b8ac)

![Screenshot 2024-11-05 at 10 03 19 AM](https://github.com/user-attachments/assets/45c1ab78-5ee2-4704-9dcd-3e278f26c4f1)

```js
{
	type: 'term',
	configKey: 'term2',
	chartType: 'barchart',
	usecase: { target: 'barchart', detail: 'term2' },
	title: 'Overlay data',
	label: 'Overlay',
	vocabApi: this.app.vocabApi,
	numericEditMenuVersion: this.opts.numericEditMenuVersion,
	defaultQ4fillTW: term0_term2_defaultQ
       processConfig(config) {
          config.settings.unit = 'abs'
      } 
},
```

Reply:
still not correct, should be `config.settings` not `this.settings` inside `processConfig()`

Reply:
I still see something wrong - does this work for you when you add an overlay term while unit is `log`? I'll be surprised if it worked that way with line 112, there's something wrong with how unit is set.

Reply:
No, this doesn't work. The only passed in config is `term2` or `term0`. That's why I used `this`. I figured you had a greater plan for the original request.

Reply:
A barchart `config.settings` can be part of the dispatch, see my `processConfig()` example above, it doesn't use `this.settings.unit == 'abs'`, it edist the config argument itself and also does not use double equal `==`.

```js
processConfig(config) {
          if (config.term2 && config.settings.unit == 'log') config.settings.unit = 'abs'
      } 
```